internal/security/proxy/container/config.go

```diff
@@ -20,8 +20,12 @@ import (
 	"github.com/edgexfoundry/edgex-go/internal/security/proxy/config"
 
 	"github.com/edgexfoundry/go-mod-bootstrap/di"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
 )
 
+// LoggingClientInterfaceName contains the name of the logging client DIC.
+var LoggingClientInterfaceName = di.TypeInstanceToName((*logger.LoggingClient)(nil))
```

Comment:
This is already defined in go-mod-bootstrap. 
https://github.com/edgexfoundry/go-mod-bootstrap/blob/master/bootstrap/container/logging.go#L24

Please use it. 
Same for Configuration.
https://github.com/edgexfoundry/go-mod-bootstrap/blob/master/bootstrap/container/configuration.go#L23

Reply:
Backed out the change.  Used suggestion for security/internal/config/main.go instead.